cmd/osm-controller/validate.go

```diff
@@ -18,6 +18,12 @@ func validateCLIParams() error {
 		}
 	}
 
+	if *certManagerKind == certmanagerKind {
```

Comment:
This is ambiguous. Would it be better to rename the cert manager type in OSM to `osmCertificateManagerKind` instead of `certManagerKind`? 

`if *osmCertificateManagerKind == certManagerKind` would make more sense.

Reply:
Sgmt!